Multisig/UI/WalletConnectV2/DefaultSignerFactory.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DefaultSignerFactory.swift
+//  Multisig
+//
+//  Created by Dirk Jäckel on 16.03.23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import CryptoSwift
+import Web3
+import Auth
+
+public struct DefaultSignerFactory: SignerFactory {
+
+    public func createEthereumSigner() -> EthereumSigner {
+        return Web3Signer()
+    }
+}
+
+public struct Web3Signer: EthereumSigner {
```

Comment:
I think this code is similar to WalletConnect example code, but it's differ from our signing logic right ?

Reply:
I will replace this code with stubs that throw exceptions so it is clear it is not used for anything.